app/Legacy/Legacy.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Legacy;
+
+use App\Models\Configs;
+use App\Models\Logs;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Session;
+
+/**
+ * Stuff we need to delete in the future.
```

Comment:
This stuff is for the users who do not use the command line.

We now have a check which has a look at the file version and compare it to the database version. If the database is behind the file, a request for migration is prompted. However this request requires admin privilege (thus admin login), as a result, we still need to support the old admin login so that the migration can be properly applied.